data/json/items/generic.json

```
@@ -107,6 +107,226 @@
+    "volume": 40,
+    "price": 100000,
+    "material": "steel",
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "quad_rotors",
+    "name": "set of rotors",
+    "name_plural": "sets of rotors",
+    "description": "A set of rotors able to lift a small drone.",
+    "weight": 100,
+    "volume": 2,
+    "price": 40000,
+    "material": [ "steel", "plastic" ],
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "android_legs",
+    "name": "set of android legs",
+    "name_plural": "sets of android legs",
+    "description": "A set of human-like legs.",
+    "weight": 10000,
+    "volume": 40,
+    "price": 100000,
+    "material": [ "steel", "plastic" ],
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "android_arms",
+    "name": "set of android arms",
+    "name_plural": "sets of android arms",
+    "description": "A set of human-like arms.",
+    "weight": 5000,
+    "volume": 20,
+    "price": 100000,
+    "material": [ "steel", "plastic" ],
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "tank_tread",
+    "name": "set of small tank tread",
+    "name_plural": "sets of small tank tread",
+    "description": "A set of small tank tread, like the one used by the beagle mini-tank.",
+    "weight": 60000,
+    "volume": 120,
+    "price": 600000,
+    "material": [ "steel", "plastic" ],
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "turret_chassis",
+    "name": "turret chassis",
+    "description": "What's left when you remove all moving parts and electronics.  It's the skeleton and armor of a turret",
+    "weight": 10000,
+    "volume": 40,
+    "price": 100000,
+    "material": [ "steel" ],
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "tripod_chassis",
+    "name": "tripod chassis",
+    "description": "What's left when you remove all moving parts and electronics.  It's the skeleton and armor of the tripod",
+    "weight": 40000,
+    "volume": "70000 ml",
+    "price": 200000,
+    "material": [ "steel" ],
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "chickenbot_chassis",
+    "name": "chicken walker chassis",
+    "description": "What's left when you remove all moving parts and electronics.  It's the skeleton and armor of the chicken walker",
+    "weight": 60000,
+    "volume": "80000 ml",
+    "price": 300000,
+    "material": [ "steel" ],
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "copbot_chassis",
+    "name": "police bot chassis",
+    "description": "What's left when you remove all moving parts and electronics.  It's the skeleton and armor of the police bot",
+    "weight": 20000,
+    "volume": "40000 ml",
+    "price": 100000,
+    "material": [ "steel" ],
+    "category": "spare_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "tankbot_chassis",
+    "name": "beagle chassis",
```

```suggestion
    "name": "Beagle chassis",
```
Also, a `plural_name` needed.